var/spack/repos/builtin/packages/visit/package.py

```diff
@@ -227,6 +228,13 @@ class Visit(CMakePackage):
     depends_on('silo+mpi', when='+silo+mpi')
     depends_on('silo~mpi', when='+silo~mpi')
 
+    depends_on('conduit@0.8.3', when='+conduit')
```

Comment:
Is `@0.8.3` a hard requirement or can we use `@0.8:`?

Also is there a version constraint on visit to support conduit?

Reply:
Sorry I intended to do `0.8.3:` -- does that sound better?

I think `0.8.2` has a build issue that we could hit. 

No version constraint for VisIt, conduit support has been there for quite some time.